docs/news.rst

```diff
@@ -3,6 +3,176 @@
 Release notes
 =============
 
+.. _release-2.10.0:
+
+Scrapy 2.10.0 (YYYY-MM-DD)
+--------------------------
+
+Highlights:
+
+-   Added Python 3.12 support, dropped Python 3.7 support.
+
+-   The add-ons framework that simplifies configuring 3rd-party components that
+    support it.
+
+-   Exceptions to retry can now be configured.
+
+-   Many fixes and improvements for feed exports.
+
+Modified requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+-   Dropped support for Python 3.7. (:issue:`5953`)
+
+-   Added support for the upcoming Python 3.12. (:issue:`5984`)
+
+-   Minimum versions increased for these dependencies:
+
+    -   lxml_: 4.3.0 → 4.4.1
+
+    -   cryptography_: 3.4.6 → 36.0.0
+
+-   ``pkg_resources`` is no longer used. (:issue:`5956`, :issue:`5958`)
+
+-   boto3_ is now recommended for exporting to S3 instead of botocore_.
+    (:issue:`5833`).
```

Comment:
```suggestion
-   boto3_ is now recommended instead of botocore_ for exporting to S3.
    (:issue:`5833`).
```

It may seem arbitrary, but it can minimize confusion, the current wording can be misread as “[…] instead of [exporting to] botocore”, however little sense that would make.